2026 Q2 community board blog post (#2712)
writethedocs/www: docs/blog/2026-Q2-community-board.rst

.. post:: Jun 22, 2026
   :tags: community board, news, report
   :author: Mikey Ariel

Write the Docs Community Board Quarterly Update Q2-2026
=======================================================

Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share our second quarterly update for 2026. 

To catch up on previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 

Q2-2026 meeting highlights
--------------------------

Our second meeting of the year covered a mix of ongoing updates and new initiatives:

**Community content spotlight**

> We reviewed shared an early draft of the new community content spotlight project, which (as an experiment in and of itself) was generated with the assistance of AI tools. We are cautiously optimistic about the possibilities of AI-assisted content for our community, but will definitely rely on human oversight to review and catch any errors (more details later in this post). 

Author: ```suggestion
We reviewed an early draft of the community content spotlight project plan, which (as an experiment in and of itself) was generated with the assistance of AI tools (more details later in this post).
```

Based on feedback from the group, we will refine the project scope to include write-ups for local WTD events. After the draft is stable, we will move forward to secure a project coordinator and volunteers who will be interfacing with the Newsletter team, since the publication at first will likely be included in our regular newsletter.

**Newsletter team growth**

> Aaron Collier reported that the newsletter team is managing the monthly publication well and has welcomed two new contributors, Felicity and Andrew. The team continues to maintain a personal editorial focus, and recently declined an applicant who submitted AI-generated content that lacked a human touch. Remember folks - Having AI skills is great, using these skills to try and pass off AI-generated content as human-generated - not great!

Author: ```suggestion
Aaron Collier reported that the newsletter team is managing the monthly publication well and has welcomed two new contributors, Felicity and Andrew. The team continues to maintain a personal editorial focus, and recently declined an applicant who submitted AI-generated content that lacked a human touch. 

Remember folks - Having AI skills is great, using these skills to try and pass off AI-generated content as human-generated - not great!
```

**From "Meetups" to "Community Groups"**

Rose Williams proposed updating the "Meetups" page title to "Community Groups", in order to be more inclusive of groups that organize local WTD events on platforms outside of Meetup.com (such as Telegram). Rose will post this proposal in the community board Slack channel to gather broader consensus before we implement the change. 

> **AI-assisted experiments and possibilities**

Author: ```suggestion
**AI-assisted experiments**
```

> Speaking of AI (who isn't these days?), we are looking into leveraging AI tools to accelerate drafting project plans and briefs for various website updates. For instance, Rose suggested a brief for a project adding YouTube links to website pages to boost SEO and cross-linking. After reviews and sign-offs by humans, these briefs could be made available as virtual Writing Day projects for community members to tackle. 

Author: ```suggestion
Speaking of AI (who isn't these days?), we are looking into leveraging AI tools to accelerate drafting project plans and content enhancements. For example, Rose suggested using AI to create a brief for a project to add YouTube links to website pages to boost SEO and cross-linking. After reviews and sign-offs by humans, these briefs could be made available as virtual Writing Day projects for community members to tackle. 
```

Other possible AI-assisted experiments include a unified branding guide, website content analysis and clean-up tasks, and general content enhancements. As always, human oversight rules the final result, but we are curious to see how these tools could help us streamline work that involves either repetitive small tasks or analyzing large data sets. 

**Slack moderation playbook initiative**

> Janine Chan suggested an initiative to codify the personal judgments made during moderation into a written playbook, which could be hosted on our website to increase transparency and trust. We discussed the feasibility of collecting raw moderation conversation data to build a dataset for AI analysis. Since we must prioritize the privacy and safety of our members is paramount, Janine will discuss this idea with the community to solicit feedback and address any data safety concerns.

Author: ```suggestion
Janine Chan suggested an initiative to codify the personal judgments made during moderation into a written playbook, which could be hosted on our website to increase transparency and trust. This process could involve collecting raw moderation conversation data to build a dataset for AI analysis. 

Since we prioritize the privacy and safety of our members, Janine will discuss this idea with the community to solicit feedback and address any data safety concerns before we proceed.
```

--------------------------

Our second meeting of the year covered a mix of ongoing updates and new initiatives:

**Community content spotlight**

We reviewed an early draft of the community content spotlight project plan, which (as an experiment in and of itself) was generated with the assistance of AI tools (more details later in this post).

Based on feedback from the group, we will refine the project scope to include write-ups for local WTD events. After the draft is stable, we will move forward to secure a project coordinator and volunteers who will be interfacing with the Newsletter team, since the publication at first will likely be included in our regular newsletter.

**Newsletter team growth**

Aaron Collier reported that the newsletter team is managing the monthly publication well and has welcomed two new contributors, Felicity and Andrew. The team continues to maintain a personal editorial focus, and recently declined an applicant who submitted AI-generated content that lacked a human touch. 

Remember folks - Having AI skills is great, using these skills to try and pass off AI-generated content as human-generated - not great!

**From "Meetups" to "Community Groups"**

> Rose Williams proposed updating the "Meetups" page title to "Community Groups", in order to be more inclusive of groups that organize local WTD events on platforms outside of Meetup.com (such as Telegram). Rose will post this proposal in the community board Slack channel to gather broader consensus before we implement the change. 

Reviewer: I don't think meetup is only for folks on meetup.com, but if we got that feedback, seems fine.

Author: That's where the proposal came from, I believe @rosewms got this feedback already

**AI-assisted experiments**

> Speaking of AI (who isn't these days?), we are looking into leveraging AI tools to accelerate drafting project plans and content enhancements. For example, Rose suggested using AI to create a brief for a project to add YouTube links to website pages to boost SEO and cross-linking. After reviews and sign-offs by humans, these briefs could be made available as virtual Writing Day projects for community members to tackle. 

Reviewer 2: Some folks for sure aren't, and won't love this tone.

Author: Right, I guess I can't be too cute about this topic and not everyone knows me enough to discern the intended tone.

> Speaking of AI (who isn't these days?), we are looking into leveraging AI tools to accelerate drafting project plans and content enhancements. For example, Rose suggested using AI to create a brief for a project to add YouTube links to website pages to boost SEO and cross-linking. After reviews and sign-offs by humans, these briefs could be made available as virtual Writing Day projects for community members to tackle. 

Author: ```suggestion
We are looking into leveraging AI tools to accelerate drafting project plans and content enhancements. For example, Rose suggested using AI to create a brief for a project to add YouTube links to website pages to boost SEO and cross-linking. After reviews and sign-offs by humans, these briefs could be made available as virtual Writing Day projects for community members to tackle. 
```

Other possible AI-assisted experiments include a unified branding guide, website content analysis and clean-up tasks, and general content enhancements. As always, human oversight rules the final result, but we are curious to see how these tools could help us streamline work that involves either repetitive small tasks or analyzing large data sets. 

**Slack moderation playbook initiative**

Janine Chan suggested an initiative to codify the personal judgments made during moderation into a written playbook, which could be hosted on our website to increase transparency and trust. This process could involve collecting raw moderation conversation data to build a dataset for AI analysis. 

> Since we prioritize the privacy and safety of our members, Janine will discuss this idea with the community to solicit feedback and address any data safety concerns before we proceed.

Reviewer 3: ```suggestion
Since we prioritize the privacy and safety of our members, Janine will discuss this idea with the moderation team to solicit feedback and address any data safety concerns before we proceed.
```
